README.md

```diff
@@ -1 +1,2 @@
 # webrainbow
+This is my project to practice HTML/CSS.
```

Comment:
There should be new line at the end of this file

Reply:
I'll do it later.